streamalert/alert_processor/outputs/aws.py

```diff
@@ -550,7 +550,7 @@ def _build_email(cls, alert, publication, creds):
         default_subject = "{} triggered alert {}".format(
             alert.rule_name, alert.alert_id
         )
-        default_body = "Please review the attached record.json"
+        default_body = "Please review the attached json record"
```

Comment:
You can over-ride this on a publisher, same applies to the record. Just drop it with `publication['@aws-ses.attach_record'] = False` and then specify it as your own attachment with your own Body

Reply:
This does seem to be a rather roundabout way of just renaming the default record, removing it and readding it with a different name.